src/styles/GlobalStyle.js

```diff
@@ -15,6 +17,17 @@ const GlobalStyle = createGlobalStyle`
     cursor: pointer;
   }
   
+=======
+    padding: 0;
+    margin: 0;
+=======
+  }
+
+  body {
+    font-family: 'SF Pro Display', sans-serif
+>>>>>>> 6540525 (DetailPageInfo 초기 완료(통신까지 진행))
+  }
+>>>>>>> 75dbd90 ([29기 임채현] 디테일 페이지 컴포넌트 초기완료)
```

Comment:
```suggestion
    padding: 0;
    margin: 0;
  }

  body {
    font-family: 'SF Pro Display', sans-serif
  }
```

conflict 해결하고 남는 부분은 꼭 제거해주세요!

Reply:
해결하였습니다!